storage/replicate_queue.go

```diff
@@ -36,6 +36,20 @@ const (
 	replicateQueueTimerDuration = 0 // zero duration to process replication greedily
 )
 
+// RebalancingOptions are configurable options which effect the way that the
```

Comment:
Ah, but you're right, I don't need it in replicate_queue.  I'll move it to allocator.